\documentclass[12pt,t]{beamer}
\usepackage{graphicx}
\setbeameroption{hide notes}
\setbeamertemplate{note page}[plain]
\usepackage{listings}

\input{header.tex}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% end of header
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% title info
\title{Dissecting and fine-mapping \emph{trans}-eQTL hotspots}
\author{Karl Broman \& Jianan Tian}
\institute{Biostatistics \& Medical Informatics, UW{\textendash}Madison}
\date{}



\begin{document}

% title slide
{
\setbeamertemplate{footline}{} % no page number here
\frame{
  \titlepage


{
\href{http://kbroman.org}{\tt \scriptsize \color{foreground} kbroman.org}
\\[-4pt]
\href{https://github.com/kbroman}{\tt \scriptsize \color{foreground} github.com/kbroman}
\hspace{1cm}
\href{https://github.com/jianan}{\tt \scriptsize \color{foreground} github.com/jianan}
\\[-4pt]
\href{https://twitter.com/kwbroman}{\tt \scriptsize \color{foreground} @kwbroman}
\\[-3pt]
\scriptsize Slides: \href{http://bit.ly/CTC2015}{\tt \scriptsize
  \color{foreground} bit.ly/CTC2015}
}

\vspace{-7mm}
\hfill \includegraphics[height=6mm]{Figs/cc-zero.png}

}

\note{
  This is a talk for the 2015 Complex Trait Community (CTC) meeting in
  Portland, Oregon, 11 June 2015.

  My graduate student, Jianan Tian, did half of the thinking and all
  of the work.

  The source for the slides is at GitHub:
  {\tt https://github.com/kbroman/Talk\_TransHotSpots}
}

}


\begin{frame}[c]{Short Course on Systems Genetics}

\small
\begin{itemize}
\item  Jackson Lab, Bar Harbor, Maine, 27 Sep {\textendash} 3 Oct, 2015

\item  \href{http://bit.ly/JaxSysGen2015}{\tt bit.ly/JaxSysGen2015}

\item QTL + gene expression + network analysis $\longrightarrow$ systems

\item 2014 instructors included:
\end{itemize}

\begin{tabular}{llll}
    \hspace{3mm} & Gary Churchill & \hspace{3mm} & Elissa Chesler        \\
&    Karl Broman          &&                        Eleanor Feingold      \\
&    \'Saunak Sen         &&                        Alan Attie            \\
&    Rafael Irizarry      &&                        Steve Horvath         \\
&    Elias Chaibub Neto   &&                        Christina Kendziorski \\
&    Greg Carter          &&                        Shawn Gomez
\end{tabular}

\note{An advertisement for a short course of considerable interest to
  many in the audience.
}
\end{frame}



\begin{frame}[c]{B6 $\times$ BTBR, \emph{Lep}$^{\text{\emph{ob}/\emph{ob}}}$}
\figw{Figs/plot-eqtl.pdf}{1.1}

\note{
    \vspace*{-12pt}
    In collaboration with Alan Attie, we've been studying
    a B6$\times$BTBR intercross, with all mice knocked out for leptin
    (and so obese), in order to understand obesity-induced diabetes.

    There are 500 intercross mice, phenotyped at a large number of
    clinical traits, and also with gene expression microarray data on
    6 tissues. These were custom two-color Agilent arrays.

    This figure shows the basic result of single-QTL genome scan for
    each expression trait, one at a time, in each tissue. Each dot is
    an inferred QTL. The y-axis is the location of the corresponding
    microarray probe, and the x-axis is the location of the QTL.

    We see a prominant diagonal, of local-eQTL, and numerous prominent
    vertical bands: ``\emph{trans}-eQTL hotspots'' where genotype at a give
    region is associated with the mRNA expression of numerous genes across
    the genome. Some of these \emph{trans}-eQTL hotspots are specific
    to a given tissue (e.g. islet chr 6) and some are seen in many
    tissues (e.g. chr 17).

    We seek to fine-map these \emph{trans}-eQTL hotspots, and to
    determine whether they involve one or multiple eQTL.
}

\end{frame}


\begin{frame}[c]{Chr 6}
\figw{Figs/chr6_lod.pdf}{1.1}

\note{
  I'm particularly interested in the locus on chromosome 6, which
  affects like 8\% of genes in pancreatic islets, but is entirely
  specific to islets.

  Each dot corresponds to the peak LOD score of a single expression
  trait: the LOD score vs the position at which it occurred. The blue
  dots are for expression traits that exist on other chromosomes; the
  brown dots are for expression traits that reside on chromosome 6.

  The local-eQTL are similar for the six tissues, but the hotspot on
  distal chr 6 in islets is seen only in islets.

  Assuming this is a single gene, how can we define a precise interval
  for the gene? We'd like to refine the localization and
  actually determine the responsible gene.
}

\end{frame}


\begin{frame}[c]{}
\centerline{\Large \ticolor Consider the non-recombinants\dots}

\note{
  Our key strategy is to focus on the non-recombinant mice (that is,
  those mice that have no recombination event in the region of this
  eQTL). For these mice, we \emph{know} their QTL genotype. We can use
  this to establish the distribution of the multivariate expression
  phenotype in each genotype group.
}

\end{frame}


\begin{frame}<handout:0>[c]{Islet c6 PCs}
\figh{Figs/islet_c6_pca_norec.pdf}{0.8}
\end{frame}

\begin{frame}[c]{Islet c6 PCs}
\addtocounter{framenumber}{-1}
\figh{Figs/islet_c6_pca.pdf}{0.8}

\note{
  We focused on the 177 microarray probes that are not on chromosome 6
  and that map to this region with LOD score $>$ 100.

  We exclude expression traits that reside on chromosome 6, thinking
  that they might be affected by separate, local-eQTL rather than the
  present locus of interest.

  We first focus on mice that had no recombination event in the 10 cM
  interval surrounding the eQTL, apply principal components analysis,
  and make a scatter plot of the first two principal components. Each
  dot is a mouse. There are three clear clusters which correspond to
  the three possible QTL genotypes.

  There were 74 mice that showed a recombination event in the
  interval; they all fall clearly into one of the three clusters.
  We can infer their eQTL genotype based on the cluster into which
  they fall.

  In this manner, the multivariate gene expression phenotype is
  converted to a co-dominant Mendelian phenotype.
}

\end{frame}


\begin{frame}<handout:0>[c]{Fine-mapping the c6 locus}
\figw{Figs/islet_c6_geno_A.pdf}{1.1}
\end{frame}

\begin{frame}<handout:0>[c]{Fine-mapping the c6 locus}
\addtocounter{framenumber}{-1}
\figw{Figs/islet_c6_geno_B.pdf}{1.1}
\end{frame}

\begin{frame}[c]{Fine-mapping the c6 locus}
\addtocounter{framenumber}{-1}
\figw{Figs/islet_c6_geno_C.pdf}{1.1}

\note{
}

\end{frame}



\begin{frame}[c]{}
\centerline{\Large \ticolor Is it one QTL?}

\note{
}

\end{frame}


\begin{frame}[c]{}
\centerline{\Large \ticolor Consider the QTL effects\dots}

\note{
}

\end{frame}



\begin{frame}[c]{eQTL effects: Islet c6}
\figw{Figs/effects_islet6.pdf}{1.2}

\note{
}

\end{frame}

\begin{frame}[c]{eQTL effects: Kidney c13}
\figw{Figs/effects_kidney13.pdf}{1.2}

\note{
}

\end{frame}

\begin{frame}[c]{eQTL effects: Islet c2}
\figw{Figs/effects_islet2.pdf}{1.2}

\note{
}

\end{frame}

\begin{frame}[c]{eQTL effects: Liver c17}
\figw{Figs/effects_liver17.pdf}{1.2}

\note{
}

\end{frame}

\begin{frame}[c]{eQTL effects: Adipose c10}
\figw{Figs/effects_adipose10.pdf}{1.2}

\note{
}

\end{frame}



\begin{frame}[c]{}
  \centering
  \Large \ticolor
  Compare the recombinants \\
  and non-recombinants.

\note{
}

\end{frame}


\begin{frame}[c]{LDA \& PCA: Islet c6}
\figw{Figs/ldapca_islet6.pdf}{1.2}

\note{
}

\end{frame}

\begin{frame}[c]{LDA \& PCA: Kidney c13}
\figw{Figs/ldapca_kidney13.pdf}{1.2}

\note{
}

\end{frame}

\begin{frame}[c]{LDA \& PCA: Islet c2}
\figw{Figs/ldapca_islet2.pdf}{1.2}

\note{
}

\end{frame}

\begin{frame}[c]{LDA \& PCA: Liver c17}
\figw{Figs/ldapca_liver17.pdf}{1.2}

\note{
}

\end{frame}

\begin{frame}[c]{LDA \& PCA: Adipose c10}
\figw{Figs/ldapca_adipose10.pdf}{1.2}

\note{
}

\end{frame}





\begin{frame}[c]{Formal test for 1 vs 2 QTL}

  \begin{itemize}
  \itemsep12pt
  \item Consider a set of traits mapping to common eQTL
  \item Multivariate QTL analysis with 1 or 2 QTL
  \item With 2-QTL model, each trait affected by one or the other QTL
    \vspace*{8pt}
    \begin{itemize}
      \itemsep8pt
      \item Order traits by estimated QTL location when considered
        separately
      \item Consider cut points of the list, assign first group to one
        QTL and second group to other.
    \end{itemize}
  \item P-value: parametric bootstrap or stratified permutation
  \end{itemize}

\note{
}

\end{frame}


\begin{frame}[c]{1 vs 2 QTL: Kidney c13}
\figw{Figs/formal_kidney13.pdf}{1.2}

\note{
}

\end{frame}

\begin{frame}[c]{1 vs 2 QTL: Islet c6}
\figw{Figs/formal_islet6.pdf}{1.2}

\note{
}

\end{frame}

\begin{frame}[c]{1 vs 2 QTL: Islet c2}
\figw{Figs/formal_islet2.pdf}{1.2}

\note{
}

\end{frame}

\begin{frame}[c]{1 vs 2 QTL: Liver c17}
\figw{Figs/formal_liver17.pdf}{1.2}

\note{
}

\end{frame}

\begin{frame}[c]{1 vs 2 QTL: Adipose c10}
\figw{Figs/formal_adipose10.pdf}{1.2}

\note{
}

\end{frame}


\begin{frame}[c]{Summary}
  \begin{itemize}
  \itemsep12pt
  \item Fine-mapping a \emph{trans}-eQTL hotspot
    \begin{itemize}
    \item Consider the non-recombinants
    \item Predict QTL genotype of recombinants \\
          $\longrightarrow$ Mendelian trait
    \item Fine-map by traditional means
    \end{itemize}
  \item Large-effect locus on chr 6
    \begin{itemize}
    \item Affects expression of $\sim$8\% of genes
    \item Effects specific to pancreatic islets
    \item Looks to be \emph{Slco1a6}
    \end{itemize}
  \item Dissecting a \emph{trans}-eQTL hotspot
    \begin{itemize}
    \item Sign of eQTL effect
    \item Degree of dominance
    \item Compare recombinants and non-recombinants
    \item Formal statistical test
    \end{itemize}
  \end{itemize}

\note{
}

\end{frame}


\begin{frame}{Acknowledgments}

{\small
  \begin{tabular}{llll}
    \hspace{5mm} & \hspace{5mm} & \hspace{5mm} & \\[-8pt]
    \multicolumn{4}{l}{\ticolor \normalsize Univ.\ Wisconsin{\textendash}Madison} \\
    & Jianan Tian                 && Brian Yandell \\
    & Alan Attie                  && Angie Oler \\
    & Mark Keller                 && Mary Rabagalia \\
    & Aimee Teo Broman            && Kathryn Schueler \\
    & Christina Kendziorski       && Donald Stapleton \\[8pt]
    \multicolumn{4}{l}{\ticolor \normalsize Univ.\ Kansas Medical Center} \\
    & Bruno Hagenbuch && Wen Zhao \\[8pt]
    \multicolumn{4}{l}{\ticolor \normalsize Merck \& Co., Inc.} \\
    & Amit Kulkarni \\[8pt]
    \multicolumn{4}{l}{\ticolor \normalsize Mt.\ Sinai} \\
    & Eric Schadt \\[8pt]
    \multicolumn{4}{l}{{\ticolor \normalsize NIH:} R01 GM074244, R01 DK066369}
\end{tabular}
}

\note{
}
\end{frame}


\begin{frame}[c]{}

\Large

Slides: \href{http://bit.ly/CTC2015}{\tt bit.ly/CTC2015}

\vspace*{-7mm}
\hspace{70mm} \includegraphics[height=5mm]{Figs/cc-zero.png}

\vspace{10mm}

\href{http://kbroman.org}{\tt kbroman.org}

\vspace{10mm}

\href{https://github.com/kbroman}{\tt github.com/kbroman}

\vspace{10mm}

\href{https://twitter.com/kwbroman}{\tt @kwbroman}


\note{
  Here's where you can find me, as well as the slides for this talk.
}
\end{frame}

\end{document}
